flow/flow.proto

```diff
@@ -43,6 +43,8 @@ message FlowLayer {
   string A = 3;
   string B = 4;
   int64 ID = 5;
+  int64 RTT = 6;
+  bool RTTSet = 7;
```

Comment:
RTT is used as temporary as well for the first ABpacket timestamp, so need a boolean to keep track when the RTT is set and valid. I agree this could be done by using FlowStat